packages/contracts/configs.js

```diff
@@ -1,3 +1,5 @@
 module.exports = {
-  REGISTRATION_FEE: 0.009 // in ether
+  SEMAPHORE_TREE_DEPTH: 4,
```

Comment:
I'm reusing the same circuit in libsemaphore with a tree depth 4, which `packages/contracts/scripts/downloadSnarks.sh` will download it from the dropbox url.